src/utils/time/time.ts

```diff
@@ -0,0 +1,27 @@
+import i18next from 'i18next';
+import moment, { Moment } from 'moment';
+
+export function getTimeDifferenceFromCurrentTimeHumanized(endTime: Moment) {

```

Comment:
I think you can remove the `CurrentTime` since the function doesn't care if it's the current time or another time the name can be a bit misleading. And I think it's best to not make the function names opinionated unless they are strictly that.